<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Habit</class>
 <widget class="QDialog" name="Habit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(206, 206, 199);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color : white;
border-width: 5px;
border-style: solid;
border-color : rgb(143, 167, 255);
background-color : rgb(143, 167, 255);
font-size : 15px;</string>
       </property>
       <property name="text">
        <string>         Habit Name   </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit">
       <property name="minimumSize">
        <size>
         <width>35</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color : white;
font-size : 15px;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color : white;
border-width: 5px;
border-style: solid;
border-color : rgb(143, 167, 255);
background-color : rgb(143, 167, 255);
font-size : 15px;</string>
       </property>
       <property name="text">
        <string>      Tracked Until</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDateEdit" name="dateEdit">
       <property name="minimumSize">
        <size>
         <width>35</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color : white;
font-size : 15px;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="minimumSize">
        <size>
         <width>95</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color : white;
border-width: 5px;
border-style: solid;
border-color : #00cc00;
background-color : #00cc00;
font-size : 15px;</string>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="styleSheet">
      <string notr="true">background-color : #CCD1D1;
border-raduis : 30px;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="minimumSize">
        <size>
         <width>95</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color : white;
border-width: 5px;
border-style: solid;
border-color : rgb(255, 78, 78);
background-color : rgb(255, 78, 78);
font-size : 15px;</string>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color : white;
border-width: 5px;
border-style: solid;
border-color : #004de6;
background-color : #004de6;
font-size : 15px;</string>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Habit</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>652</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>689</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
